cmd/diff/main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/pkg/diff"
+)
+
+var (
+	color       = flag.Bool("color", false, "color")
+	contextSize = flag.Int("context", 3, "context size")
```

Comment:
What do you think about `-u` instead of `-context`, to match traditional `diff`?

Reply:
Done.